### ISSUE TITLE
Adonis new project : Cannot find module 'adonis-fold'

### ISSUE THREAD

**Issue by turlup** [adonisjs/adonis-framework#324]
Hi,

On my Mac I have 3.0 Adonis project who works very well. I want create new project with last version 3.1 and I have a problem with a fresh installation :

**After "adonis new api" in console :**

```
MacBook:api guid$ npm install
npm ERR! git clone --template=/Users/guid/.npm/_git-remotes/_templates --mirror git://github.com/crypto-utils/keygrip.git /Users/guid/.npm/_git-remotes/git-github-com-crypto-utils-keygrip-git-f7b41c2f305cfa7030d7da9e64af9790aa3be93a-8e08e45288d43e48751cc9b3c03e56a8: Cloning into bare repository '/Users/guid/.npm/_git-remotes/git-github-com-crypto-utils-keygrip-git-f7b41c2f305cfa7030d7da9e64af9790aa3be93a-8e08e45288d43e48751cc9b3c03e56a8'...
npm ERR! git clone --template=/Users/guid/.npm/_git-remotes/_templates --mirror git://github.com/crypto-utils/keygrip.git /Users/guid/.npm/_git-remotes/git-github-com-crypto-utils-keygrip-git-f7b41c2f305cfa7030d7da9e64af9790aa3be93a-8e08e45288d43e48751cc9b3c03e56a8: fatal: read error: Operation timed out
npm ERR! Darwin 16.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.0.0
npm ERR! npm  v3.0.0
npm ERR! code 128

npm ERR! Command failed: git clone --template=/Users/guid/.npm/_git-remotes/_templates --mirror git://github.com/crypto-utils/keygrip.git /Users/guid/.npm/_git-remotes/git-github-com-crypto-utils-keygrip-git-f7b41c2f305cfa7030d7da9e64af9790aa3be93a-8e08e45288d43e48751cc9b3c03e56a8
npm ERR! Cloning into bare repository '/Users/guid/.npm/_git-remotes/git-github-com-crypto-utils-keygrip-git-f7b41c2f305cfa7030d7da9e64af9790aa3be93a-8e08e45288d43e48751cc9b3c03e56a8'...
npm ERR! fatal: read error: Operation timed out
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/guid/Documents/MYSITE/api/npm-debug.log
MacBook:api guid$ npm run dev

> adonis-app@3.1.0 dev /Users/guid/Documents/MYSITE/api
> nodemon --watch app --watch bootstrap --watch config --watch .env -x "node --harmony_proxies" server.js

[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/guid/Documents/MYSITE/api/app/**/* /Users/guid/Documents/MYSITE/api/bootstrap/**/* /Users/guid/Documents/MYSITE/api/config/**/* .env
[nodemon] starting `node --harmony_proxies server.js`
module.js:338
    throw err;
    ^

Error: Cannot find module 'adonis-fold'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/guid/Documents/MYSITE/api/bootstrap/http.js:14:14)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
[nodemon] app crashed - waiting for file changes before starting...
```

**My package.json :**

```json
{
  "name": "adonis-app",
  "version": "3.1.0",
  "main": "index.js",
  "scripts": {
    "dev": "nodemon --watch app --watch bootstrap --watch config --watch .env -x \"node --harmony_proxies\" server.js",
    "start": "node --harmony_proxies server.js",
    "lint": "standard"
  },
  "author": "",
  "license": "MIT",
  "description": "",
  "private": true,
  "dependencies": {
    "adonis-ace": "^3.0.4",
    "adonis-auth": "^1.0.4",
    "adonis-commands": "^2.1.2",
    "adonis-fold": "^3.0.3",
    "adonis-framework": "^3.0.3",
    "adonis-lucid": "^3.0.8",
    "adonis-middleware": "^1.0.8",
    "standard": "^8.4.0",
    "youch": "^1.0.1"
  },
  "standard": {
    "global": [
      "use",
      "make"
    ]
  },
  "autoload": {
    "App": "./app"
  },
  "devDependencies": {
    "nodemon": "^1.11.0",
    "standard": "^8.3.0"
  }
}
```

I have test a lot of versions of node(4.x.x)+npm(3.x.x) before post (same issues).
Any ideas ? Thank you.

**Comment by thetutlage** [comment]
I believe `npm install` failed due to some reasons. Can u remove `node_modules` and re-install everything.

```bash
rm -rf node_modules && npm install
```

**Comment by thetutlage** [comment]
Closing since not directly related to AdonisJs and npm was unable to install the modules.

**Comment by ashinzekene** [comment]
But it doesn't still work !!!

**Comment by RomainLanz** [comment]
Hey @ashinzekene 👋 

May you open another issue please?
